src/map/map.cpp

```diff
@@ -51,7 +52,19 @@ bool Map::extractMap(const std::string& identifier) const {
 	if (boost::filesystem::exists(identifier)) {
 		return true;
 	}
-	
+
+	if (CURL *curl = curl_easy_init()) {
+		SPDLOG_INFO("Downloading world.zip to world folder");
+		FILE *fp = fopen("data/world/world.zip", "wb");
+		curl_easy_setopt(curl, CURLOPT_FOLLOWLOCATION, 1L);
+		curl_easy_setopt(curl, CURLOPT_URL, "https://www.dropbox.com/s/nmc8w82one8mmp9/world.zip?dl=1");
```

Comment:
As I suggested in the discord group, the map file is already in the release, feel free to update the link

```suggestion
		curl_easy_setopt(curl, CURLOPT_URL, "https://github.com/opentibiabr/otservbr-global/releases/download/patch-v1.3.1/world.zip");
```